examples/svelte-apollo/nhost/migrations/default/1651998909892_create_table_public_article/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "public"."article" (
```

Comment:
I would suggest changing the table name to plural as the table contains multiple articles.

Reply:
Done